src/modules/expense/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { styled, Theme } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
+const StyledDivHeader = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  width: '100%',
+  alignItems: 'center',
+  justifyContent: 'space-around',
+  marginTop: '20px',
+}));
+
+export const Header = () => {
```

Comment:
This is how I'd recommend to change the Header component:
```
import ArrowBackIcon from '@mui/icons-material/ArrowBack';
import Stack from '@mui/material/Stack';
import { styled, Theme } from '@mui/material/styles';
import Typography from '@mui/material/Typography';

const StyledDivHeader = styled(`div`)(({ theme }: { theme: Theme }) => ({
  marginTop: theme.spacing(2),
  marginBottom: theme.spacing(2),
  position: 'relative',
}));

export const Header = () => {
  return (
    <StyledDivHeader>
      <ArrowBackIcon
        fontSize='large'
        sx={{ color: 'white', position: 'absolute' }}
      />
      <Stack direction='row' justifyContent='center'>
        {/* Replace with a shared Typography component */}
        <Typography variant='h5' component='h5' sx={{ color: 'white' }}>
          Expense
        </Typography>
      </Stack>
    </StyledDivHeader>
  );
};
```

In reality, I think it will become a shared component very soon. It will accept a title prop and display different title. It may also have different button on the left and right, e.g. https://www.figma.com/file/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?type=design&node-id=722-3211&mode=design&t=A600otlgrCckPHJQ-0

Reply:
At the beginning I used Header Component but then I realize Title1 component for shared titles. So I created Title3 component to use shared component except using Header. I mean I already using shared title component for header section

Reply:
Replied here: https://github.com/frontend-from-0/montra-client/pull/8#discussion_r1573912000